cmake/package_deb.cmake

```diff
@@ -66,7 +66,7 @@ set(CPACK_COMPONENTS_ALL applications doc library share ${THIRDPARTY_COMPONENT_G
 
 ## Autogeneration with SHLIBDEPS will add to this variable. For now we include most things statically and require the standard Qt package only.
 ## (only available in Ubuntu >=17.10). For older Ubuntu, dependencies can be installed from a thirdparty repo.
-set(CPACK_DEBIAN_PACKAGE_DEPENDS "qtbase5-dev (>= 5.7.0) | qt57base | qt58base | qt59base | qt510base | qt511base, libqt5svg5 (>= 5.7.0) | qt57svg | qt58svg | qt59svg | qt510svg | qt511svg")
+set(CPACK_DEBIAN_PACKAGE_DEPENDS "qtbase5-dev (>= 5.7.0) | qt57base | qt58base | qt59base | qt510base | qt511base, libqt5svg5 (>= 5.7.0) | qt57svg | qt58svg | qt59svg | qt510svg | qt511svg | libqt5sql5-sqlite | libsqlite3-dev")
```

Comment:
do we still need the package names for older ubuntus from third-party repos?

Reply:
14.04 and 16.04 are still technically supported by Canonical, if it's not adding any overhead for us I'd suggest keeping them until those are EOL.

Reply:
ok

Reply:
I wonder how the plugin is called on the beineri repo tho